truss/constants.py

```diff
@@ -33,6 +33,9 @@
 SUPPORTED_PYTHON_VERSIONS = {"3.8", "3.9", "3.10", "3.11"}
 
 TRTLLM_PREDICT_CONCURRENCY = 512
+TRTLLM_MIN_MEMORY_REQUEST = "24Gi"
```

Comment:
nit, these two memory settings seem related just differet format. May be refactor the value 24 out to separate variable.